mlir/test/Dialect/ArmSME/vector-ops-to-sme.mlir

```diff
@@ -578,3 +578,99 @@ func.func @transpose_f64(%arg0: vector<[2]x[2]xf64>) {
   "prevent.dce"(%0) : (vector<[2]x[2]xf64>) -> ()
   return
 }
+
+//===----------------------------------------------------------------------===//
+// vector.outerproduct
+//===----------------------------------------------------------------------===//
+
+// -----
+
+// CHECK-LABEL: @vector_outerproduct_masked_f64
```

Comment:
nit: everywhere else the tests are ordered smallest to largest (f16, bf16, f32, f64). It could just as well be largest to smallest, that's not particularly important, but I think consistency is. I'd appreciate it if we could be consistent, it makes reviewing easier.

Reply:
Done (other than keeping the main example first)